docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -115,7 +115,7 @@ something provider-specific. The tabs below will contain a notice about what fla
 on `kubeadm init` are required.
 - Unless otherwise specified, kubeadm uses the default gateway's network interface 
 to advertise the master's IP. If you want to use a different network interface, specify 
-`--apiserver-advertise-address=<ip-address>` argument to `kubeadm init`.
+`--apiserver-advertise-address=<ip-address>` argument to `kubeadm init`. To configure IPV6 for apiserver, this argument should be specified with IPV6 enclosed in square brackets `--apiserver-advertise-address=[ipv6-address]` 
```

Comment:
Brackets are not needed when specifying an `apiserver-advertise-address` IPv6 address. Brackets should only ever be required when specifying an IPv6 tuple, i.e. `[fd00:db01::100]:8080`. 

Instead of the docs you added to line 118, consider the following "To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you must specify an IPv6 `--apiserver-advertise-address`".

nit: Please replace all references of IPV6 with IPv6 (lowercase v)